modules/sno-du-disabling-ntp.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-disabling-ntp_{context}"]
+= Disabling Network Time Protocol (NTP)
+
+After the system is configured for Precision Time Protocol (PTP), you need to
+remove NTP from impacting the system clock.
+
+.Procedure
+
+* No configuration changes are needed. Use the provided settings:
++
+[source,yaml]
```

Comment:
I'm not sure about this one. I don't see it on my 4.10 cluster.

Reply:
@imiller0 - Can you verify this?

Reply:
There is a bit of work in progress around how to handle this particular CR. For now I would recommend keeping this CR here, but it will likely change to a much simpler implementation "soon". When that occurs we can update the docs to reflect the change.  
@yliu127 this would need to be explicitly included via a PolicyGenTemplate in order to disable chronyd (which any system with PTP running would need).

Reply:
ACK